examples/README.md

```diff
@@ -10,3 +10,12 @@ This directory holds several examples using rules_xcodeproj. To open an example,
 
 * **sanitizers**
     <br> Contains targets to test Sanitizers in BwB mode. The main purpose is to make sure sanitizers in BwB mode work and give UI feedback the same way sanitizers do in BwX builds.
+
+## External
+
+Here are some examples outside of this repository that show how to use
+rules_xcodeproj in various ways:
```

Comment:
Might want to say something about this not necessarily being up to date since they're permalinks, or to check the latest versions in those project's `main` branch.